src/app/section-main/main-information/main-information.component.html

```diff
@@ -0,0 +1,27 @@
+<section class="main-information">
+  <p class="main-information__slogan">Долго, дорого, богато!</p>
+  <a to="/catalog">
+    <button class="main-button" value="">каталог изделий</button>
+  </a>
+  <hr class="main-information__hr" />
+  <div class="brends">
+    <div class="brends__item">
+      <img src="./assets/images/brend.png" alt="brend" />
+    </div>
+    <div class="brends__item">
+      <img src="./assets/images/brend.png" alt="brend" />
+    </div>
+    <div class="brends__item">
+      <img src="./assets/images/brend.png" alt="brend" />
+    </div>
+    <div class="brends__item">
+      <img src="./assets/images/brend.png" alt="brend" />
+    </div>
+    <div class="brends__item">
+      <img src="./assets/images/brend.png" alt="brend" />
+    </div>
+    <div class="brends__item">
+      <img src="./assets/images/brend.png" alt="brend" />
+    </div>
```

Comment:
Use ngFor directive to display repeated data.

Reply:
fix